SwiftWisdom/Core/StandardLibrary/Array/Array+Utilities.swift

```diff
@@ -122,11 +123,11 @@ extension Sequence where Iterator.Element: Equatable {
         return buffer
     }
     
-    public func ip_countOf(_ element: Iterator.Element) -> Int {
+    public func ip_countOf(element: Iterator.Element) -> Int {
         return self.filter { $0 == element } .count
     }
     
-    public func ip_containsAll<T: Sequence where T.Iterator.Element == Iterator.Element>(all: T) -> Bool {
+    public func ip_containsAll<T: Sequence>(all: T) -> Bool where T.Iterator.Element == Iterator.Element {
```

Comment:
Maybe this should just read like `ip_contains(all: ...)`... just trying to say remove the all from the func name since it will be in the first param. Alternatively hide the external parameter name so it's `ip_containsAll(...)` I'm not sure which or style favors.